extensions/eventtap/init.lua

```diff
@@ -177,29 +177,41 @@ function module.rightClick(point, delay)
     module.event.newMouseEvent(module.event.types["rightMouseUp"], point):post()
 end
 
---- hs.eventtap.middleClick(point[, delay])
```

Comment:
@mongito or are you referring to leaving in the documentation for middleClick?  My thought was keep it as a valid function, hence setting middleClick = otherClick, so existing code doesn't break, but encourage future use of the correct function by documenting only the new function since it's more in line with apple's naming conventions.